xds/internal/balancer/weightedtarget/weightedtarget_config_test.go

```diff
@@ -24,32 +24,32 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"google.golang.org/grpc/balancer"
 	internalserviceconfig "google.golang.org/grpc/internal/serviceconfig"
-	_ "google.golang.org/grpc/xds/internal/balancer/cdsbalancer"
+
+	_ "google.golang.org/grpc/xds/internal/balancer/lrs"
```

Comment:
Sorry didn't notice last time around. A small comment explaining why the blank import is done.

Reply:
Done